### PR TITLE
Use nameof keyword instead of hard coded parameter names

### PR DESCRIPTION
**Pull request by walterlv**
It's better for method refactoring.

### CODE REVIEW

File: src/Microsoft.DotNet.Wpf/src/System.Xaml/System/Xaml/Context/ObjectWriterFrame.cs [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 namespace MS.Internal.Xaml.Context
 {
-    [DebuggerDisplay("{ToString()}")]
+    [DebuggerDisplay("{" + nameof(ToString) + "()}")]
```
[the remark this answers is not in the document]
Comment (walterlv) [reply src/Microsoft.DotNet.Wpf/src/System.Xaml/System/Xaml/Context/ObjectWriterFrame.cs]: You're right, and I didn't notice it before.

**Comment by walterlv** [comment]
@karelz I've removed the unnecessary `DebuggerDisplay`.

**Comment by karelz** [comment]
@walterlv I originally suggested to remove the attribute in separate PR. I let WPF team have the final word if they are ok to have it in the same PR (I don't have strong preference).
You missed couple of other `ToString` instances in your PR

**Comment by walterlv** [comment]
@karelz OK, I'll check others.

**Comment by walterlv** [comment]
@karelz I removed all changes for DebuggerDisplay, keep them as originals.

**Comment by rladuca** [comment]
I ported all the shared changes over to our private repos so that we can merge this.

Thanks @walterlv!

**Pull request merged**